plasmapy/formulary/mathematics.py

```diff
@@ -90,6 +91,21 @@ def Fermi_integral(
         raise TypeError(f"Improper type {type(x)} given for argument x.")
 
 
+def Chandrasekhar_G(x):
+    """
+    The Chandrasekhar function G(x) goes as 2x / 3 \sqrt{\pi} at low x and 0.5
+    x^{-2} describes the drag on a particle by collisions with a Maxwellian
```

Comment:
```suggestion
    The Chandrasekhar function :math:`G(x)` goes as :math:`2x / 3 \sqrt{π}` at low :math:`x`
    and :math:`0.5 x^{-2}` describes the drag on a particle by collisions with a Maxwellian
```
Hopefully I got the back ticks (or were they regular ticks?) right.

Reply:
Likely! Like I said, that docstring is a WIP still :)